timers.js

```diff
@@ -1,4 +1,36 @@
+var resetB=document.querySelector('#reset');
+var startB=document.querySelector('#start');
+var pauseB=document.querySelector('#pause');
+var seconds;
+var timerId;
 
+resetB.addEventListener("click", resetTimer);
+startB.addEventListener("click", startTimer);
+pauseB.addEventListener("click", stopTiming);
+function startTimer(){
+  startB.removeEventListener('click', clearInterval(timerId));
+  var head1=document.querySelector('#timer');
 
+    timerId=window.setInterval(updateTime, 500);
+    console.log("time started");
+  }
 
+function updateTime(){
+  seconds=seconds+1;
```

Comment:
This is the root of your NaN problem. Can you see why the first time you run this code the value of seconds becomes NaN. Remember that NaN is the result of performing an operation which should be performed on numbers on values of a different type